source/less/blocks/advantages.less

```
@@ -171,14 +176,43 @@
 }
 
 .advantages__inner-title {
+  position: relative;
   font-size: 18px;
   font-weight: 700;
   margin: 0;
   margin-bottom: 20px;
 
+  &::before {
+    content: "";
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    top: -40%;
+    left: -50px;
+  }
+
+  &--housing::before {
+    background: url("../img/icon-house.svg") no-repeat center;
+    background-size: cover;
```

здесь contain должен быть. Нам всегда нужно чтобы иконка показывалась целиком, а не обрезалась, как это может сделать свойсво cover. 
```suggestion
    background-size: contain;
```